06 - Calendar/dycalendar.js

```diff
@@ -148,19 +148,6 @@
         div.setAttribute("class", "dycalendar-header");
         div.setAttribute("data-option", JSON.stringify(option));
 
-        //prev button
-        if (option.prevnextbutton === "show") {
-            elem = document.createElement("span");
-            elem.setAttribute("class", "dycalendar-prev-next-btn prev-btn");
-            elem.setAttribute("data-date", option.date);
-            elem.setAttribute("data-month", option.month);
-            elem.setAttribute("data-year", option.year);
-            elem.setAttribute("data-btn", "prev");
-            elem.innerHTML = "&lt;";
-            //add prev button span to header div
-            div.appendChild(elem);
-        }
-
         //month span
```

Comment:
File:05 - Random Password Generator/main.js 

@@ -40,10 +40,18 @@ const words = ['seat', 'pen', 'broad', 'vapor', 'ocean',
     
 
 function randomNumber (l) {
-    return Math.floor(Math.random() * l);
+    let y = Math.floor(Math.random() * l);
+
+    let z = Math.floor(Math.random() * y);
+    return y;
 }
 
 function randomWord () {
     const number = randomNumber(words.length);
     return words[number];
 }
+
+function randomWord222 () {
+    const number = randomNumber(words.length);
+    return words[number];
+}

#######
Comment:
#######
I think this should be `randomWord222`
#######
File:06 - Calendar/dycalendar.js